Sources/ApplicativeRouter/Combinators.swift

```diff
@@ -174,7 +180,7 @@ public let put = method(.put)
 
 // MARK: - Private
 
-private func typeKey<A>(_ a: A) -> String {
+private func typeKey(_ a: Any) -> String {
```

Comment:
Without this, the printer prints `:any` for `queryParams` values. With it, it prints `:__nscfnumber` for int values...progress?